docs/src/pages/guides/composition/ComponentProperty.js

```diff
@@ -72,30 +71,27 @@ ListItemLinkShorthand.propTypes = {
 function ComponentProperty(props) {
   const { classes } = props;
 
-  // Use NoSsr to avoid SEO issues with the documentation website.
```

Comment:
Google Bot uses its JavaScript VM eventually. It doesn't solve the core problem. A 301 is more appropriate, we can simplify the demo.